kadai1/kimuson13/conversion/conversion.go

```diff
@@ -0,0 +1,93 @@
+package conversion
+
+import (
+	"errors"
+	"fmt"
+	"image"
+	"image/gif"
+	"image/jpeg"
+	"image/png"
+	"os"
+	"path/filepath"
+	"strings"
+)
+
+var supportedextension = []string{"jpeg", "jpg", "png", "gif"}
+
+//Info is a type for os.fileinfo
+type Info = os.FileInfo
```

Comment:
```suggestion
type Info os.FileInfo
```

Gopher道場の資料と標準パッケージを見る限りこちらのほうがいいかもしれません

Reply:
ユーザー定義型の使い方がよくわかっていなかったため、苦肉の策としてos.FileInfoをユーザー定義型にしていました。
```type Info os.FileInfo```　にするとエラーがでてビルドできなくなるので、```=```を使ってました。
pull requestを送った後にgopher道場の続きのインターフェイスについての資料や動画を見て、レシーバ等の理解が深まったので、ユーザー定義型を一新してみました。
他の箇所も直したので、また何かありましたらコメントお願いします。